Source/relay/TourGuide/Items of the Month/2022/Autumnaton.ash

```diff
@@ -57,7 +57,7 @@ void IOTMAutumnatonGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEnt
 		description.listAppend("Visit mid underground for +1 autumn item (Cyrpt zone, Daily Dungeon?)");
```

Comment:
fun fact, i do not believe you can send it to DD after you've done the 15th room of the DD for that day. i tested this in a recent run and i may have just been tired but i think it locks it out. kinda funny.